src/pudl/scripts/dbt_helper.py

```diff
@@ -108,23 +108,10 @@ def add_column_tests(self, column_tests: dict[str, list]) -> "DbtSource":
 
         return self.model_copy(update={"columns": list(columns.values())})
 
-    @staticmethod
-    def get_row_count_test_dict(table_name: str, partition_column: str):
-        """Return a dictionary with a dbt row count data test encoded in a dict."""
-        return [
-            {
-                "check_row_counts_per_partition": {
-                    "table_name": table_name,
-                    "partition_column": partition_column,
-                }
-            }
-        ]
-
     @classmethod
-    def from_table_name(cls, table_name: str, partition_column: str) -> "DbtSchema":
+    def from_table_name(cls, table_name: str) -> "DbtSchema":
         """Construct configuration defining table from PUDL metadata."""
         return cls(
```

Comment:
This claims to be returning a `DbtSchema` but is actually returning a `DbtTable`